api/layer/layer.go

```diff
@@ -307,26 +308,24 @@ func TimeNow(ctx context.Context) time.Time {
 // Owner returns owner id from BearerToken (context) or from client owner.
 func (n *layer) Owner(ctx context.Context) user.ID {
 	if bd, ok := ctx.Value(api.BoxData).(*accessbox.Box); ok && bd != nil && bd.Gate != nil && bd.Gate.BearerToken != nil {
-		return bearer.ResolveIssuer(*bd.Gate.BearerToken)
+		return bd.Gate.BearerToken.ResolveIssuer()
 	}
 
 	var ownerID user.ID
-	if err := user.IDFromKey(&ownerID, n.gateKey.PublicKey().Bytes()); err != nil {
-		panic(fmt.Errorf("id from key: %w", err))
-	}
+	ownerID.SetScriptHash(n.gateKey.PublicKey().GetScriptHash())
 
 	return ownerID
 }
 
 func (n *layer) prepareAuthParameters(ctx context.Context, prm *PrmAuth, bktOwner user.ID) {
+	prm.PrivateKey = &n.gateKey.PrivateKey
```

Comment:
Why are you moving this? Previous logic was different.

Reply:
previous logic uses or bearer token or private key. The bearer token had more weight. Right now this `prm.PrivateKey` is used to sign client requests as explicit parameter